### DISCUSSION TITLE
[Scam Alert] reavester

### DISCUSSION THREAD

**Discussion by micro73** [Webamoozcom/warning-list#148] [Investigation بررسی]
پروژه reavester
آدرس سایت : https://reavester.com/

توضیحات سایت :
ReaVester  یک مدیر دارایی نهادی با چندین دهه سابقه در امور مالی، بانکداری و نوآوری است. تمرکز اصلی ما بر روی ارزهای دیجیتال، تجارت، سرمایه گذاری سرمایه گذاری، بانکداری و نوآوری است.
سابقه ما از سال 2005 با بیش از 5 شرکت توسعه یافته از صنایع صفر تا چند میلیون دلاری است. ما در دارایی‌های دیجیتال و شرکت‌های بلاک چین سرمایه‌گذاری کرده‌ایم و طیف کاملی از قرار گرفتن در فضا را در اختیار سرمایه‌گذاران قرار می‌دهیم. ما به استارت‌آپ‌ها کمک کردیم تا رشد کنند، بزرگ‌آپ‌ها به اهداف خود دست یابند و برای سرمایه‌گذاران و معامله‌گران خود استقلال مالی ایجاد کردیم.
پلن های سرمایه گذاری : 

![13](https://user-images.githubusercontent.com/103103225/183832201-9e3b948a-5569-4e7e-894b-f55ad192f043.jpg)

بررسی های انجام شده : 
طبق گفته خودشون از سال 2005 فعالیت دارند درحالی که سایت تاسیس شده سال 2022 هست 
اطلاعات افراد گرداننده و پشت پروژه طبق معمول مخفی است و مشخص نیست چه افرادی با چه هویتی هستند .

![12](https://user-images.githubusercontent.com/103103225/183832307-b77b1a88-e4d0-4e92-becd-303f4cf5b889.jpg)

بررسی های سایت اسکم ادوایزر : (( گویای همه چیز هست )) 

![10](https://user-images.githubusercontent.com/103103225/183832331-ef7c3b38-59b6-4911-8f3e-55f52b8d6fef.jpg)
![11](https://user-images.githubusercontent.com/103103225/183832339-5e2502ed-28b2-4931-8cb3-a092103bbf66.jpg)

ترجمه ماشینی : 
مالک وب سایت با استفاده از یک سرویس پولی هویت خود را در WHOIS مخفی می کند
این وب سایت بازدید کننده زیادی ندارد
ما بسیاری از وب‌سایت‌های با رتبه پایین را در یک سرور پیدا کردیم
چندین اسپمر و کلاهبردار از یک ثبت کننده استفاده می کنند
ما خدمات ارزهای دیجیتال را شناسایی کردیم که می توانند ریسک بالایی داشته باشند
روش های پرداخت ناشناس در این سایت شناسایی شد
این وب سایت به تازگی ثبت شده است.

-	در سایت فقط میتوانید وارد پنل کاربری شوید و برای ثبت نام از نفر معرفی کننده باید لینک دریافت کنید که این نشان میدهد سیستم بر اساس شبکه سازی پایه گذاری شده است و با افزایش افراد ورودی طبیعتا درامد شما افزایش خواهد یافت
-	حق عضویت سایت 30 دلار است که اولین پکیج توضیح داده شده است و سودی از این پکیج نمیگیرید و صرفا حق عضویت است .
-	سیستم اهراز هویت در سایت موجود است که بعد از دریافت لینک از نفر دعوت کننده و تکمیل ثبت نام باید احراز هویت کنید که دادن اطلاعات به این سایت هشدار دارد.
-	در صورت لغو جایگاه 31درصد مبلغ مانده اکانت شما و 3 درصد کارمز از شما برداشت میکنند و دکمه کنسل تا زمانی فعال است که 66 درصد از پولتان باقی مانده باشد.
-	لینک های سایت هیچ کدام کار نمیکند و هیچ شبکه اجتماعی ندارد

![14](https://user-images.githubusercontent.com/103103225/183832401-ba8267a2-def9-4d7d-8f12-917ed2795e54.jpg)
![18](https://user-images.githubusercontent.com/103103225/183832406-d6c1cc39-0c34-4ce9-ba85-e281f7bca10b.jpg)
![15](https://user-images.githubusercontent.com/103103225/183832748-320e0ce6-d07f-492f-934a-241b4343e207.jpg)

سایت از سال 2022 شروع به فعالیت کرده ولی رودمپ صحبت 2015 میکند و سال 2022 هیچ کدام از رودمپ انجام نشده است .
خودشون در ابتدای سایت گفتن فعالیت از 2015 شرو شده در جایی دیگه میگن سال تاسیس 2017 هست و دیدیم سایت 2022 راه اندازی شده است .

![16](https://user-images.githubusercontent.com/103103225/183832784-1c5c10f3-55b8-4c80-91fd-dc61c4149249.jpg)
![17](https://user-images.githubusercontent.com/103103225/183832792-210cc5c6-9173-4188-8a6b-89a869cf59fe.jpg)

فکر نکنم نیاز به توضیحات و بررسی بیشتری باشد همه چی مشخص و گویاست . مراقب باشید.

**Comment by micro73** [comment]
شواهد گویای این مطلب هست که این مجموعه  در حال راه اندازی سایت دیگر  تحت عنوان tnsinvestment  هستند

http://www.tnsinvestment.com

![2022-08-20_11-23-23](https://user-images.githubusercontent.com/103103225/185733137-44ce11ee-1a1a-4859-809e-c45a0e1dee18.jpg)

**Comment by iranhoshdar** [comment]
**### گروه  reavester  که اکنون به tnsinvestment تغییر نام داده و پایگاه شروع آنها در استانبول ترکیه واقع است شروع به عضو گیری دوباره نموده اند**

  

![image](https://user-images.githubusercontent.com/112306866/187060893-408a1e19-d12a-4bc7-ac8e-a276f39218e0.png)

   

**###  این گروه با ادعای شرکتی ۱۷ سال تجربه در زمینه سرمایه گذاری دامین خود را در 11 August 2022  ثبت کرده یعنی ۱۷ روز قبل از نوشتن این متن ( تصویر زیر )**
  


![image](https://user-images.githubusercontent.com/112306866/187060932-fe2f3448-de5c-46f2-adba-c67e7e8c3294.png)



![image](https://user-images.githubusercontent.com/112306866/187060946-a6732bd3-eb55-4c0b-adfa-bd10fe09b73f.png)


![image](https://user-images.githubusercontent.com/112306866/187060809-d34d93d2-7a7d-4585-bba2-6235723da67e.png)

**Comment by iranhoshdar** [comment]
نماهایی از پنل کاربری

پلان های سرمایه گذاری

![image](https://user-images.githubusercontent.com/112306866/187061120-d85c9399-6f65-4d23-b863-360e9b942e16.png)


![image](https://user-images.githubusercontent.com/112306866/187061165-ba4a364b-b590-4202-9b02-62081278c0a9.png)

**Comment by iranhoshdar** [comment]
مطالب آپدیت شده در کانال مخصوص این سیستم کلاهبردار گذاشته شده است 
آدرس کانال تلگرام : https://t.me/tnsinvestment_reavester
